### PR TITLE
Backport 0.15 pr25

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
backporting #10179 causes the build to fail. It appears to be caused by some boost problems... I tried to troubleshoot it but wasn't able to resolve it

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
added more backports since this was a small PR, quite concerned about 10695

File: test/functional/bip65-cltv-p2p.py [hunk 1]
```diff
@@ -4,172 +4,162 @@
...
+    tx.deserialize(BytesIO(hex_str_to_bytes(signresult['hex'])))
+    return tx
+
+class BIP65Test(BitcoinTestFramework):
 
     def __init__(self):
         super().__init__()
         self.num_nodes = 1
-        self.extra_args = [['-whitelist=127.0.0.1', '-blockversion=3', '-dip3params=9000:9000']]
+        self.extra_args = [['-promiscuousmempoolflags=1', '-whitelist=127.0.0.1']]
```
Comment (UdjinM6) [inline test/functional/bip65-cltv-p2p.py]:
```suggestion
        self.extra_args = [['-promiscuousmempoolflags=1', '-whitelist=127.0.0.1', '-dip3params=9000:9000']]
```

File: test/functional/bipdersig-p2p.py [hunk 2]
```diff
@@ -39,144 +36,122 @@ def unDERify(tx):
...
+    tx.deserialize(BytesIO(hex_str_to_bytes(signresult['hex'])))
+    return tx
+
+class BIP66Test(BitcoinTestFramework):
 
     def __init__(self):
         super().__init__()
         self.num_nodes = 1
-        self.extra_args = [['-dip3params=9000:9000']]
+        self.extra_args = [['-promiscuousmempoolflags=1', '-whitelist=127.0.0.1']]
```
Comment (UdjinM6) [inline test/functional/bipdersig-p2p.py]:
```suggestion
        self.extra_args = [['-promiscuousmempoolflags=1', '-whitelist=127.0.0.1', '-dip3params=9000:9000']]
```

**Comment by PastaPastaPasta** [comment]
was missing 10192, that should add `promiscuousmempoolflags` I think

**Comment by PastaPastaPasta** [comment]
doesn't build due to witness conflicts. I'm not sure how to resolve them.

**Comment by UdjinM6** [comment]
pls see https://github.com/UdjinM6/dash/commits/pr3080

**Comment by UdjinM6** [comment]
Looks good imo, will test a bit.

File: src/policy/policy.cpp [hunk 3]
```diff
@@ -16,18 +16,20 @@
...
-    // i.e. 182 * 3 = 546 duffs with default -minrelaytxfee = minRelayTxFee = 1000 duffs per kB.
+    // "Dust" is defined in terms of dustRelayFee,
+    // which has units duffs-per-kilobyte.
+    // If you'd pay more in fees than the value of the output
+    // to spend something, then we consider it dust.
+    // A typical spendable txout is 34 bytes big, and will
+    // need a CTxIn of at least 148 bytes to spend:
+    // so dust is a spendable txout less than
+    // 182*dustRelayFee/1000 (in duffs).
+    // 546 duffs at the default rate of 3000 sat/kB.
```
Comment (thephez) [inline src/policy/policy.cpp]: Shouldn't this be duffs/kB instead of "sat/kB"?

Comment (PastaPastaPasta) [inline src/policy/policy.cpp]:
```suggestion
    // 546 duffs at the default rate of 3000 duff/kB.
```

**Pull request merged**